File: Serverside/aspnet-core/src/Homies.RealEstate.Core/Server/ThuocTinh.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using Abp.Domain.Entities.Auditing;
using Abp.Domain.Entities;

namespace Homies.RealEstate.Server
{
    [Table("ThuocTinhs")]
    public class ThuocTinh : Entity
    {

        [Required]
        [StringLength(ThuocTinhConsts.MaxTenThuocTinhLength, MinimumLength = ThuocTinhConsts.MinTenThuocTinhLength)]
        public virtual string TenThuocTinh { get; set; }

        [Required]
        [StringLength(ThuocTinhConsts.MaxKieuDuLieuLength, MinimumLength = ThuocTinhConsts.MinKieuDuLieuLength)]
        public virtual string KieuDuLieu { get; set; }

        [Required]
        [StringLength(ThuocTinhConsts.MaxTrangThaiLength, MinimumLength = ThuocTinhConsts.MinTrangThaiLength)]
        public virtual string TrangThai { get; set; }

    }
}

File: Clientside/flutter/lib/models/post/propertiesforpost/ThuocTinh.dart
class ThuocTinh {
  String tenThuocTinh;
  String kieuDuLieu;
  String trangThai;
  int id;
  ThuocTinh({
    this.tenThuocTinh,
    this.kieuDuLieu,
    this.trangThai,
    this.id,
  });

  factory ThuocTinh.fromMap(Map<String, dynamic> json) => ThuocTinh(
    tenThuocTinh: json["thuocTinh"]["tenThuocTinh"],
    kieuDuLieu: json["thuocTinh"]["kieuDuLieu"],
    trangThai: json["thuocTinh"]["trangThai"],
    id: json["thuocTinh"]["id"],
  );

  Map<String, dynamic> toMap() => {
    "tenThuocTinh": tenThuocTinh,
    "kieuDuLieu": kieuDuLieu,
    "trangThai": trangThai,
    "id": id,
  };

}
